frontend/static/js/lib.js

```diff
@@ -138,10 +138,6 @@ function formatPercentage(v) {
   return v + "%";
 }
 
-function numberWithCommas(x) {
-  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
-}
-
```

Comment:
this was added in the previous commit and is removed here again. It's probably a part of #93 ?